recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
   skip: true  # [win]
```

Comment:
I guess we need to change the meta.yaml?

i.e. windows is ok now, py>38 should still be skipped, and then our setup.py has the following restrictions for our deps:

```python
    if sys.version_info[:2] < (3, 6):
        # pinned to support deprecated python versions
        install_requires = [
                'numpy>=1.13.3,<1.17',
                'biopython>=1.71,<1.77',
                'joblib>=0.12,<0.15.0']
    else:
        install_requires = [
                'numpy>=1.13.3',
                'biopython>=1.71',
                'joblib>=0.12']


    install_requires.extend([
        'networkx>=1.0',
        'GridDataFormats>=0.4.0',
        'six>=1.4.0',            # to support Py 2
        'mmtf-python>=1.0.0',
        'scipy>=1.0.0',
        'matplotlib>=1.5.1,<3.4', # to support Py <3.7
        'mock',
        'tqdm>=4.43.0',
        'funcsigs',              # to support Py 2
    ])
```